docs/core/tools/dotnet-migrate.md

```diff
@@ -20,6 +20,8 @@ dotnet migrate [-h|--help]
 
 ## Description
 
+This command is deprecated. It is no longer available in and after .NET Core 3.x SDK. It can only migrate a Preview 2 .NET Core project to a 1.x .NET Core project which is currently out of support.
```

Comment:
This makes line 48 redundant. Should you remove that one then?

Reply:
I moved line 42 up there. And removed the next paragraph. Looks better?